db/db_blob_index_test.cc

```diff
@@ -398,6 +398,26 @@ TEST_F(DBBlobIndexTest, Iterate) {
     verify(15, Status::kOk, get_value(16, 0), get_value(14, 0),
            create_blob_iterator, check_is_blob(false));
 
+    // Iterator with blob support and using seek.
```

Comment:
how do you make sure `FindValueForCurrentKeyUsingSeek` is called in this test?

Reply:
From this [line](https://github.com/facebook/rocksdb/blob/3b3b96d6933c64b9c47dba19020e0dcce214c668/db/db_iter.cc#L724), we know that when `max_skip_==0` backward iteration must call `FindValueForCurrentKeyUsingSeek` to retrieve value. I verify the case by running the test without the fix, and it failed.